src/core/public/ui_settings/ui_settings_api.ts

```diff
@@ -90,6 +88,21 @@ export class UiSettingsApi {
     });
   }
 
+  public async getAll() {
+    return this.sendRequest('GET', '/api/opensearch-dashboards/settings', undefined);
+  }
+
+  public async getWithScope() {
```

Comment:
No need for getWithScope I think if scope is already a property of the UISettingAPI instance.